insights/parsers/up2date_log.py

```diff
@@ -0,0 +1,38 @@
+"""
+up2date Logs -  Files ``/var/log/up2date``
+==========================================
+
+Modules for parsing the content of log file ``/var/log/up2date`` in sosreport archives of RHEL.
+
+.. note::
```

Comment:
This `Note` is duplicated with the one inner the class,  better remove this one.